Directory.Build.props

```diff
@@ -68,4 +68,13 @@
     <_RunJNIEnvGen Condition=" '$(JIBuildingForNetCoreApp)' == 'True' ">dotnet "$(_JNIEnvGenPath)"</_RunJNIEnvGen>
     <_RunJNIEnvGen Condition=" '$(JIBuildingForNetCoreApp)' != 'True' ">$(Runtime) "$(_JNIEnvGenPath)"</_RunJNIEnvGen>
   </PropertyGroup>
+  
+  <!-- Add Roslyn analyzers NuGet to all projects -->
+  <ItemGroup>
```

Comment:
Don't item groups actually belong in [`Directory.Build.targets`](https://docs.microsoft.com/en-us/visualstudio/msbuild/customize-your-build?view=vs-2019)?

Reply:
I went with the guidance here: https://docs.microsoft.com/en-us/visualstudio/msbuild/customize-your-build?view=vs-2019#choose-between-adding-properties-to-a-props-or-targets-file.  

Specifically:
```
Include items in .props files (conditioned on a property). All properties are considered before any item, so user-project property customizations get picked up, and this gives the user's project the opportunity to Remove or Update any item brought in by the import.
```
I added the `$(DisableRoslynAnalyzers)` to allow opting out of running the analyzers.